examples/images_contours_and_fields/plot_streamplot.py

```diff
@@ -16,63 +16,58 @@
 """
 import numpy as np
 import matplotlib.pyplot as plt
-import matplotlib.gridspec as gridspec
+
 
 w = 3
 Y, X = np.mgrid[-w:w:100j, -w:w:100j]
 U = -1 - X**2 + Y
 V = 1 + X - Y**2
 speed = np.sqrt(U**2 + V**2)
 
-fig = plt.figure(figsize=(7, 9))
-gs = gridspec.GridSpec(nrows=3, ncols=2, height_ratios=[1, 1, 2])
+fig, ax = plt.subplots(3, 2, figsize=(7, 9), height_ratios=[1, 1, 2])
```

Comment:
```suggestion
fig, axs = plt.subplots(3, 2, figsize=(7, 9), height_ratios=[1, 1, 2])
```
Since it’s a collection. We use `ax` only for a single Axes.